cloudflare/src/endpoints/workers/mod.rs

```diff
@@ -1,27 +1,35 @@
 use crate::framework::response::ApiResult;
 
+use chrono::offset::Utc;
+use chrono::DateTime;
 use serde::Deserialize;
 
 mod create_route;
+mod create_secret;
 mod delete_route;
+mod delete_secret;
 mod list_routes;
+mod list_secrets;
 
 pub use create_route::{CreateRoute, CreateRouteParams};
+pub use create_secret::{CreateSecret, CreateSecretParams};
 pub use delete_route::DeleteRoute;
+pub use delete_secret::DeleteSecret;
 pub use list_routes::ListRoutes;
+pub use list_secrets::ListSecrets;
 
 /// Workers KV Route
 /// Routes are basic patterns used to enable or disable workers that match requests.
 /// https://api.cloudflare.com/#worker-routes-properties
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct WorkersRoute {
-    /// Namespace identifier tag.
-    pub id: String,
-    /// The basic pattern that should map to the script
-    pub pattern: String,
-    /// Name of the script to apply when the route is matched.
-    /// The route is skipped when this is blank/missing.
-    pub script: Option<String>,
+  /// Namespace identifier tag.
```

Comment:
looks like your linter is messing with tabs. tab size should be 4 spaces. seems to be only this file? idk, make sure you check all files.